src/Sarif.Converters/SemmleQlConverter.cs

```diff
@@ -14,9 +14,9 @@ namespace Microsoft.CodeAnalysis.Sarif.Converters
     /// <summary>
     /// Converts a log file from the Semmle format to the SARIF format.
     /// </summary>
-    internal class SemmleQLConverter : ToolFileConverterBase
+    public class SemmleQLConverter : ToolFileConverterBase
     {
-        // Semmle logs are CSV files.
+        // Semmle logs are CSV files with inconsistent contents
```

Comment:
If you're going to add this comment, a "for example" would help. Now when I read the code I'm worried about how they're inconsistent.